pkgs/by-name/gh/ghostty/package.nix

```diff
@@ -60,6 +60,12 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-AHI1Z4mfgXkNwQA8xYq4tS0/BARbHL7gQUT41vCxQTM=";
   };
 
+  # Avoid using runtime hacks to help find X11
+  postPatch = lib.optionalString (appRuntime == "gtk") ''
+    substituteInPlace src/apprt/gtk/x11.zig \
+      --replace-warn 'std.DynLib.open("libX11.so");' 'std.DynLib.open("${lib.getLib libX11}/lib/libX11.so");'
+  '';
+
```

Comment:
I'm going to submit a PR soon to add a build option to specify the location of `libX11.so`. That way in the future we don't need to patch the code.